ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -27,8 +27,8 @@
  */
 #include <wiredtiger.h>
 #include <wiredtiger_ext.h>
-#include <sys/stat.h>
 #include <fstream>
+#include <experimental/filesystem>
```

Comment:
We're going to need extra logic (maybe in a header file somewhere) to determine how to import this library. We'll need to add some checks to see if this needs to be imported under the experimental namespace (for older compiler and C++ STDLs) or if that's not necessary. A potential solution is [here](https://stackoverflow.com/questions/53365538/how-to-determine-whether-to-use-filesystem-or-experimental-filesystem)